quarto/intro.qmd

```diff
@@ -8,7 +8,77 @@
 
 ### Render vs preview (Guilherme / Estelle)
 
-## Computations (Mahesh / Guilherme)
+## Computations
+
+Quarto lets you perform computations within your notebook. This is typically
+done using code blocks denoted by three backticks followed by the language 
+you're using in curly brackets.
+
+```
+```{{python}}
+1 + 1
+```
+```
+
+Quarto supports computations in several languages:
+- [Python](https://quarto.org/docs/computations/python.html)
+- [R](https://quarto.org/docs/computations/r.html)
+- [Julia](https://quarto.org/docs/computations/julia.html)
+- [Observable](https://quarto.org/docs/computations/ojs.html)
+
+Additional languages can also be supported through other Jupyter kernels (see Engines below).
+See this page for a list of [Jupyter kernels](https://gist.github.com/chronitis/682c4e0d9f663e85e3d87e97cd7d1624).
+
+The languages and packages used in your computations must be available in your `render` environment, and 
+are often installed through other means, for example using `conda` or a container platform.
+
+Packages and variables created within one code block are also accessible from other code blocks, including
+inline code blocks. For example, here we create a figure and reference the value in the caption.
+
+````
+
+:::{#fig-plot-alt}
+
+```{r}
+x <- 1:10
+y <- x^2
+
+plot(x, y)
+```
+
+A plot of $x$ against it's square (n = `{r} length(x)`).
+:::
+
+````
+
+The output of computations can be controlled using [execution options](https://quarto.org/docs/computations/execution-options.html).
+These can be set for the whole document in the yaml front-matter at the top of the document, e.g.,
+
+````
+---
+title: My Experiment
+execute:
+  echo: false
+---
+````
+
+Alternatively, execution options can be specified within each code block, e.g.,
+
+````
+```{python}
```

Comment:
I'm not sure how you want this displayed, but on my side for the code block to show up I need to double curly brace the language declaration {{python}}

Reply:
I could have sworn the documentation had it like this, but I can't find it now. 
Anyway, checked it all with `quarto preview` and it renders now how I want.